scripts/get_bank_registry_be.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+import json
+import xlrd
+import requests
+
+URL = "https://www.nbb.be/doc/be/be/protocol/r_fulllist_of_codes_current.xlsx"
+
+
+def process():
+    registry = []
+    skip_names = ["NAV", "VRIJ", "NAP", "NYA", "VRIJ - LIBRE", "-"]
+    book = xlrd.open_workbook(file_contents=requests.get(URL).content)
+
+    try:
+        sheet = book.sheet_by_index(0)
+        for row in list(sheet.get_rows())[2:]:
+            bank_code, bic, name, second_name = row[:4]
+            if bic.value.upper() in skip_names:
+                continue
+            if name.value != "":
+                registry.append({
+                    "country_code": "BE",
+                    "primary": True,
+                    "bic": bic.value.upper().replace(" ", ""),
+                    "bank_code": bank_code.value,
+                    "name": name.value,
+                    "short_name": name.value,
+                })
+            else:
+                registry.append({
+                    "country_code": "BE",
+                    "primary": True,
+                    "bic": bic.value.upper().replace(" ", ""),
+                    "bank_code": bank_code.value,
```

Comment:
This is all copy-pasted! Now there are two locations that you have to change if you want to adapt any of these values in the registry. There is convenient pattern that you can use for your kind of problem takes advantage of the fact that empty strings evaluate to `False` in a boolean context.

```python
s1 = "foo"
s2 = ""
s3 = s2 or s1 # s3 == "foo"
```

So in your case you could write
```python
{
   ...
   "name": name.value or short_name.value
   ...
}
```

Reply:
Yes, I felt it was a bit too easy to do it like that but I wasn't aware of that handy solution. I will fix it in a second. I'm sorry again. I feel really bad about trying to help. 😢